pages/news/2020-08-02-portable-mixxx-setup.html

```diff
@@ -0,0 +1,85 @@
+This "Traktor ready" controller had been given to me over the previous owner's frustrations with the lack of mappings for any other software than Traktor.
+
+### Putting one and two together
+
+Being both a controller and audio interface, the Akiyama Pulsar reduced the overhead created by multiple USB devices (?) allowing the Raspberry Pi to run Mixxx smoothly with < 20ms latency and no underruns.
+Note, this is without any optimizations to the Raspbian OS (no realtime kernel, no JACK) so there is possibly room for improvement!
+
+As there was no mapping for the controller I set out to create my own.
+Most buttons are easily mapped usng the GUI, but the jogwheels can be problematic.
+Starting from an existing community contributed mapping script that did not intimidate me (Icon iDJ) and by launching Mixxx with the `--controllerDebug` argument I was able to at least get the jogwheels to be somewhat reactive.
+
+### Skinning the fat
+
+To make the setup extra portable I dressed my Raspberry Pi in a 2.8" Touch Screen that is long out of production.
+Being rather impractical, I would suggest getting a larger screen.
+Nevertheless, you will need a skin that accomodates the data you are interested in, while being readable on a small screen.
+
+Again I started of from an existing skin, an older "minimal" skin found on in an old Mixxx install (< 2.0.0) only to find out there is no scaling, only absolute values for size and position as well a different naming convention for buttons, making it incompatible with newer skins and mappings ("library" verus "playlist" for example).
+
+The skins provided by the later Mixxx distributions seemed rather indimidating at first but soon made me realized they are far more flexible than they look.
+Next to dynamic scaling using `(e,e)` units the newer default skins are somewhat modular.
+Departing from LateNight i simply deleted the style xml files for the buttons/sections I did not need, ending up with a minimal two deck view of waveforms, track name, bpm and using a leftover key on my controller to show and hide the fullscreen library!
+
+The paths where Mixxx stores it's default skins and controller mappings are different on various operating Systems.
+Working with both OSX and Raspbian this was somewhat confusing at times.
+To add to this, the directory of the built-in skins/mappings is not the same as the user skins/mappings.
+As soon as you edit a default mapping for example, it gets copied to your "user" settings directory.
+Keep this in mind (and just edit/copy your custom mapping directly to your "user" settings folder).
+
+### In short
+
+* Get a Raspberry Pi 3 Model B or newer
+* Find a USB DJ controller with built-in sound-card.
+* Get a powerbank that can deliver the right amount of current.
+  The specifications advertised/printed most electronics don't seem to tell much, there is only one way to find out...try it out and let the [community](https://mixxx.zulipchat.com/) know!
```

Comment:
advertised about most
I disagree with this assertion: the spec printed on the device tells you the maximum amount of current it will draw, so your power supply needs to be able to deliver at least that amount. Sometimes you might get away with something slightly less, but do you really want to take that chance if you're DJing live?

Reply:
I think the issue is that the RPi can draw more than 5V when running CPU intensive tasks while USB devices are connected. I think that's why the official power supply delivers 5.1V instead of the usual 5V of USB power supplies (and I think that cheap models can gave just 4.9V or something like that).

Reply:
Should we rephrase it?